data/examples/declaration/value/function/operator-chains-out.hs

```diff
@@ -0,0 +1,245 @@
+module A (a) where
+
+-- Template
+-- x :: Int
+-- x =
+--   1 op_a
+--   2 op_a
+--   3 op_a
+--   4 op_b
+--   5 op_a
+--   6 op_a
+--   7 op_a
+--   8
+
+-- Right chain, normal case, 1 operator type
+a :: Int
+a =
+  1
+    ++ 2
+    ++ 3
+    ++ 4
+
+-- Right chain, : case, 1 operator type
+b :: Int
+b =
+  1
+    : 2
+    : 3
+    : 4
+    : 5
+    : 6
+    : 7
+    : 8
+
+-- Right chain, $ case, 1 operator type
+c :: Int
+c =
+  1 $
+    2 $
+      3 $
+        4 $
+          5 $
+            6 $
+              7 $
+                8
+
+-- Right chain, normal case, 2 operators with p(a) == p(b)
+e :: Int
+e =
```

Comment:
What about:

```
e =
  1
    ^^ 4
    ^ 5
    ^^ 8
```

Reply:
I don't really understand your suggestion here. Do you mean that we should make the example shorter?

Reply:
Yes.